content/millennium/r4/financial/general/charge-item.md

```diff
@@ -55,19 +55,19 @@ The following fields are returned if valued:
 All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/StructureDefinition/{id}`
 
  ID                     | Value\[x] Type                    | Description
```

Comment:
These column titles look like they need there width changed to align with the changes made below.

Reply:
99c460943bac20003ddaf8ed1d5e505d211da8e7